### PR TITLE
fix flaky integration test

### PR DESCRIPTION
**Pull request by dimkouv**

### CODE REVIEW

File: core/services/ocr2/plugins/ccip/integration_test.go [hunk 1]
```diff
@@ -277,6 +277,9 @@ merge [type=merge left="{}" right="{\\\"%s\\\":$(link_parse), \\\"%s\\\":$(eth_p
 			t.Logf("deleting job %d", id)
 			err = node.App.DeleteJob(context.Background(), id)
 			require.NoError(t, err)
+
+			ccipTH.Source.Chain.Commit()
+			ccipTH.Dest.Chain.Commit()
```
Comment (mateusz-sekara) [inline core/services/ocr2/plugins/ccip/integration_test.go]: Hmm, interesting. Why do we need to progress with chain when all the operations in the for loop are related do jobs crud?

Comment (dimkouv) [reply core/services/ocr2/plugins/ccip/integration_test.go]: from my understanding there are some evm calls involved during the process, and the fresh state is required but I didn't dive deeper
```
https://github.com/smartcontractkit/ccip/blob/03ca68df5f946efa1cc07610148a8ebbc815630a/core/services/ocr2/delegate.go#L262
```

Comment (dimkouv) [reply core/services/ocr2/plugins/ccip/integration_test.go]: but can actually be moved out of the loop

**Review by makramkd — APPROVED** [review]
Works for me locally with this change when it was failing pretty persistently before.

**Review by mateusz-sekara — APPROVED** [review]

**Pull request merged**